tests/gamma/csm_observability_test.py

```diff
@@ -201,9 +203,11 @@ def initKubernetesClientRunner(self, **kwargs) -> KubernetesClientRunner:
     # each run().
     def initKubernetesServerRunner(self, **kwargs) -> GammaServerRunner:
         return super().initKubernetesServerRunner(
-            enable_csm_observability=True,
-            csm_workload_name=CSM_WORKLOAD_NAME_SERVER,
-            csm_canonical_service_name=CSM_CANONICAL_SERVICE_NAME_SERVER,
+            deployment_args=ServerDeploymentArgs(
```

Comment:
I think you missed this part. With this change, lines 35 and 104 won't be needed too.

```suggestion
            deployment_args=gamma_server_runner.ServerDeploymentArgs(
```

Reply:
Done